samza-core/src/main/java/org/apache/samza/config/ApplicationConfig.java

```diff
@@ -0,0 +1,58 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.samza.config;
+
+/**
+ * Accessors for configs associated with Application scope
+ */
+public class ApplicationConfig extends MapConfig {
+  /**
+   * processor.id is equivalent to containerId in samza. It is a logical identifier used by Samza for a processor.
```

Comment:
I think we are supporting it. 
>  I don't think the processor.id is equivalent to containerId.
I hope you are also referring to containerId as the logical containerId used by Samza, which is different from the physical containerId provided by Yarn. Why do you think they are not equivalent? If we modify our Yarn-based implementation today to bring up more than one instance of SamzaContainer within the same JVM, it would be equivalent to the use-case we are offering now. Are you saying they are not equivalent because we have a 1:1 mapping between JVM and SamzaContainer today?